SIT.Manager.Avalonia/Models/ManagerConfig.cs

```diff
@@ -28,7 +28,7 @@ public partial class ManagerConfig : ObservableObject
         [ObservableProperty]
         public bool _lookForUpdates = true;
         [ObservableProperty]
-        public LocalizationModel _currentLanguageSelected = new() { FullNameLanguage = CultureInfo.CurrentCulture.NativeName, ShortNameLanguage = CultureInfo.CurrentCulture.Name };
+        public string _currentLanguageSelected = CultureInfo.CurrentCulture.Name;
```

Comment:
This falls back to the users current culture info, but if the user's culture doesn't match any of the localizations we have the selected field will be blank (In my case this defaults to en-GB, since that doesn't match any localizations we have the selected item becomes blank)

Reply:
Yeah, I forgot about that issue will fix it.